README.md

```diff
@@ -873,7 +873,9 @@ TODO-B: - no incautious use of malloc/sprintf (the language has no direct MM)
 TODO: - no use of sudo, gksu, pkexec, or LD_LIBRARY_PATH (usage is OK inside
 TODO:   tests)
 TODO: - no use of user nobody
+TODO:   (consider at least `grep` for it and `find . -user nobody`)
 TODO: - no use of setuid
+TODO:   (consider at least `grep` for it and `find . -perm -4000`)
```

Comment:
I can't recall how find modes work, would `-6000` look for both setuid and setgid files/directory or would it look for files/directories that have *both* setuid and setgid?

Reply:
It would look for an AND, so only for those having both set.
This would be right:
`find . \( -perm -4000 -o -perm -2000 \)`

I'll adapt accordingly and with some outcome of the discussion in our meeting.